packages/ia-components/sandbox/theatres/components/audio-player/players_by_type/archive-audio-jwplayer-wrapper.jsx

```diff
@@ -105,11 +124,31 @@ class ArchiveAudioPlayer extends Component {
     }
   }
 
+  /**
+   * Post JWPlayer registration handler - returns optional registration callback
+   *
+   * Currently, this is where we support external ability to set URL
+   * through Internet Archive's JWPlayer Wrapper
+   */
+  postRegistration() {
+    const { onRegistrationComplete, jwplayerInfo, needsURLSettingAccess = false } = this.props;
+    const { identifier = '' } = jwplayerInfo;
+
+    if (onRegistrationComplete && needsURLSettingAccess) {
+      const externallySyncURL = function externallySyncURL (identifier, trackNumber) {
+        const playlistIndex = trackNumber - 1 || 0;
```

Comment:
SUGGESTION: I don't believe this Boolean operator will do anything. If `trackNumber` is 1, then it's `0 || 0` which is still... `0`.

Reply:
yes, agree. it was a defensive reflex.